test/cadet_web/controllers/auth_controller_test.exs

```diff
@@ -26,7 +25,7 @@ defmodule CadetWeb.AuthControllerTest do
   alias Cadet.Auth.Guardian
   alias CadetWeb.AuthController
 
-  @token if System.get_env("TOKEN"), do: System.get_env("TOKEN"), else: "token"
+  @code System.get_env("CODE") || "code"
```

Comment:
Not sure why you need the `|| "code"` still.

Reply:
we need it as a placeholder if code is not given during the test so the tests will still run!